cps/main.c

```diff
@@ -511,10 +513,12 @@ cps_proc(void *arg)
 		 * transmit to respective Gatekeeper interfaces.
 		 */
 		process_egress(cps_conf, front_iface, front_kni,
-			cps_conf->tx_queue_front);
+			cps_conf->tx_queue_front,
+			cps_conf->back_max_pkt_burst);
```

Comment:
Shouldn't  it be `cps_conf->front_max_pkt_burst` instead of `cps_conf->back_max_pkt_burst` here?